Source/Editor/Options/InputOptions.cs

```diff
@@ -56,6 +56,10 @@ public sealed class InputOptions
         [EditorDisplay("Common"), EditorOrder(190)]
         public InputBinding SelectAll = new InputBinding(KeyboardKeys.A, KeyboardKeys.Control);
 
+        [DefaultValue(typeof(InputBinding), "Ctrl+Shift+A")]
```

Comment:
What about `Escape`?

Reply:
There were a couple of discussions about this in discord. You can review them yourself and choose what you see fit.
![image](https://github.com/FlaxEngine/FlaxEngine/assets/32620310/62b07de0-a825-46c2-a89c-922ba7767c04)
![image](https://github.com/FlaxEngine/FlaxEngine/assets/32620310/49b45745-c65e-404b-a78b-503b2c98aff0)
![image](https://github.com/FlaxEngine/FlaxEngine/assets/32620310/64db9cd6-27e5-4f0b-bf6b-76b0e684cfd2)
![image](https://github.com/FlaxEngine/FlaxEngine/assets/32620310/d2d691a6-559e-4359-b1da-920a02f12ccb)

Sorry for having to post it as images here, but the discussion is very very fragmented lol

Reply:
okay makes sense, then let's leave it as it is with option to rebind by user 😄